.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,35 @@
+{
+  "image": "ubuntu:22.04",
+  "containerEnv": {
+    "DOTNET_CLI_TELEMETRY_OPTOUT": "true",
+    "DOTNET_NOLOGO": "true",
+    "DOTNET_SKIP_FIRST_TIME_EXPERIENCE": "true",
+    "DOTNET_USE_POLLING_FILE_WATCHER": "true",
+    "NUGET_XMLDOC_MODE": "skip"
+  },
+  "forwardPorts": [
+    2375,
+    2376
+  ],
```

Comment:
do we need this?

Reply:
No, not really. I added it to get rid of the pop up message.

Reply:
I removed `forwardPorts`.